plugins/tpm/fu-tpm-device.c

```diff
@@ -249,7 +249,7 @@ fu_tpm_device_setup (FuDevice *device, GError **error)
 		return FALSE;
 	if (!fu_plugin_tpm_get_uint32 (ctx, TPM2_PT_FIRMWARE_VERSION_2, &version2, error))
 		return FALSE;
-	version_raw = ((guint64) version1) << 32 | ((guint64) version2);
+	version_raw = ((guint64) GUINT32_FROM_BE (version1)) << 32 | ((guint64) GUINT32_FROM_BE (version2));
```

Comment:
But we've already done `GUINT32_FROM_BE` in `fu_plugin_tpm_get_uint32()` so this means it's undoing the byte-swaps and operating on raw bytes. Is that expected? I think we just want to remove *both* `GUINT32_FROM_BE` if that's the case.

Reply:
Ah yes; I agree.

Reply:
I've done that now.